frontend/packages/console-dynamic-plugin-sdk/src/extensions/pvc.ts

```diff
@@ -1,6 +1,5 @@
-import { Extension } from '@console/plugin-sdk/src/typings/base';
-import { ExtensionDeclaration, CodeRef } from '../types';
-import { ExtensionCommonK8sResource as PVCResource } from '../utils/common';
```

Comment:
there were issues with single `.d.ts` file genetated by `DtsBundleWebpack` when `foo as bar` syntax is used. The `PVCResource` was missing in the generated file